src/lib/auth.ts

```diff
@@ -427,12 +427,12 @@ export const authenticate = (appMode: AppMode, urlParams?: QueryParams) => {
   });
 };
 
-export const getProblemIdForAuthenticatedUser = (bearerToken: string, urlParams?: QueryParams) => {
+export const getProblemIdForAuthenticatedUser = (rawPortalJWT: string, urlParams?: QueryParams) => {
   return new Promise<string|undefined>((resolve, reject) => {
     if (urlParams && urlParams.offering) {
       superagent
       .get(urlParams.offering)
-      .set("Authorization", `Bearer ${bearerToken}`)
+      .set("Authorization", `Bearer/JWT ${rawPortalJWT}`)
```

Comment:
@dougmartin mentioned this shouldn't be necessary...but I can't get Teacher Dashboard reports working on staging unless I make this change. They don't currently work on master.